packages/components/psammead-script-switch/src/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_WHITE, C_EBON } from '@bbc/psammead-styles/colours';
+import { string, shape, node } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const SIZE_ABOVE_600PX = '3rem'; // 48px
+const SIZE_BELOW_600PX = '2.5rem'; // 40px
+
+const getDimensions = size => `
+  height: ${size}
+  width: ${size}
+`;
+
+const StyledLink = styled.a`
```

Comment:
There should be padding between the text and the border:
![script-switch](https://user-images.githubusercontent.com/46446236/66053510-5e994180-e52a-11e9-8ec0-e109005d61f5.png)

Reply:
I was looking at the first design with Chinese switch script. I din't see the padding. But I can see it on the other design. Thanks for catching this.

Reply:
True 🤔, always refer to Zeplin to see the most updated version.

Reply:
I think that the width of the component is given by the padding.

Reply:
@DenisHdz I don't think so. That would mean `(16 * 2) + (1.25(line-height) * 16)` which results into a height and width of `52`.

Reply:
I believe the height should not be proportional to the width as you can see in the Serbian designs which have `48px/40px` for the height, but not for the width.